spec/models/shopify_import/client_spec.rb

```diff
@@ -1,57 +1,92 @@
 require 'spec_helper'
 
 RSpec.describe ShopifyImport::Client, type: :model do
-  describe '.instance' do
-    context 'with valid credentials' do
-      let(:credentials) { { api_key: 'foo', password: 'baz', shop_domain: 'test_shop.myshopify.com' } }
-      let(:client) { described_class.instance }
-      let(:site) { 'https://foo:baz@test_shop.myshopify.com/admin' }
+  describe '#get_connection' do
+    let(:client) { described_class.instance }
 
-      before { client.get_connection(credentials) }
+    after { ShopifyAPI::Base.clear_session }
 
-      it 'creates connection to shopify api' do
-        expect(ShopifyAPI::Base.site.to_s).to eq site
+    context 'with api_key and password' do
+      let(:credentials) do
+        {
+          api_key: '0a9445b7b067719a0af024610364ee34', password: '800f97d6ea1a768048851cdd99a9101a',
```

Comment:
@vfonic Do we really want to have data for real shop in specs? Even if it is test store?